layouts/download/section.html

```diff
@@ -1,17 +1,31 @@
 {{ define "main" }}
-{{ partial "download/hero.html" . }}
-{{ with .Content }}
-<section class="container pt-lg-5">
-  <div class="row">
-    <div class="col-12 col-lg-5 pr-lg-5">
-      {{ partial "download/download.html" . }}
-    </div>
-    <div class="col-12 col-lg-7">
-      {{ . }}
+<div class="scalable download-hero d-flex align-items-center">
+  <div class="container text-light">
+    <div class="row pt-5 pb-5">
+      <div class="col col-12 text-center">
+        <h1 class="title">
+          {{ .Title }}
+        </h1>
+        <h4 class="pb-1">
+          {{ .Content }}
+        </h5>
+      </div>
     </div>
   </div>
-</section>
-{{ end }}   
-{{ partial "download/getting-started.html" . }}  
+</div>
+<div id="os-selector" class="os-selector">
+  <div class="os-selector-row">
+    <a id="os-windows" class="os-selector-button active" href="#windows" data-os="windows">Windows</a>
+    <a id="os-linux" class="os-selector-button" href="#linux" data-os="linux">Linux</a>
+  </div>
+</div>
+<div id="content-windows" class="os-content active">
+  {{ partial "download/content-windows.html" . }}
+  {{ partial "download/getting-started-windows.html" . }}
+</div>
+<div id="content-linux" class="os-content">
+  {{ partial "download/content-linux.html" . }}
+  {{ partial "download/getting-started-linux.html" . }}
+</div>
```

Comment:
Another option is to have the card title read `Install O3DE as a binary` or however we choose to message it, keeping the card content the same, but changing the link destination **only** in the tab toggle.

Reply:
I guess it looks weird cause it's the only thing that changes down there, but the actual content changes may make it look ok?

I'm open to changes to the UX as long as we agree on the direction :)

Reply:
@sptramer how about the call to action button on top? The windows button said "Download O3DE" and the linux button showed "Download Debian Package" should we also make these two consistent?

Reply:
We want to be pretty clear about what Linux platforms are supported. Here's actually what I'd suggest:

* Windows: `Download O3DE Installer`
* Linux: `Download O3DE .deb Package`

Keeps the up front wording consistent and ensures that it's clear about what's being pulled from the binary repo.

Reply:
Note that this PR is just for the technical change to add the tabs, we haven't added the final copy yet. The plan right now is to submit this to staging so that Doug can access it and make the copy and requirement changes independently.
If the issue is that we don't want the second row to also change with the tab, I can just pull it out but I was under the impression we wanted to give different directions for Linux in those boxes?

Reply:
Agreed with @AMZN-daimini . I will leave the final copy to @erickson-doug.